.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,21 @@
+{
+    "configurations": [
+        {
+            "name": "Linux",
+            "includePath": [
+                "${workspaceFolder}/**",
+                "${workspaceFolder}/soccer/src/soccer",
+                "${workspaceRoot}/**",
+                "/opt/ros/foxy/include/**",
+                "/home/dwu359/robocup-software/soccer/src/soccer"
+            ],
+            "defines": [],
+            "compilerPath": "/usr/bin/clang",
+            "cStandard": "c17",
+            "cppStandard": "c++14",
+            "intelliSenseMode": "linux-clang-arm64",
+            "configurationProvider": "ms-vscode.cmake-tools"
+        }
+    ],
+    "version": 4
+}
```

Comment:
Make sure that the changes that you make for your vscode configs are not pushed to the main branch.